src/reducers/functionList.js

```diff
@@ -0,0 +1,15 @@
+
+export function functionListReducer (state = {}, action) {
+
+  if(action.type ==='FUNCTION_LIST_LOAD' & action.status === 'SUCCESS' ) 
+    return Object.assign({}, state, { list: action.functionList, status: action.status })
+
+  if(action.type ==='FUNCTION_LIST_LOAD' & action.status !== 'SUCCESS' ) 
+    return Object.assign({}, state, { status: action.status })
```

Comment:
I think I'd prefer to concatenate these 2 if statements both to reduce checking the same thing twice, and make it a little more readable:
```
if (action.type === 'FUNCTION_LIST_LOAD') {
    if (action.status === 'SUCCESS) {
        return ...
    } else {
        return ...
    }
```